index.html

```diff
@@ -9,45 +9,241 @@
+     </li>
+     <li>
+       <a class="Tags" href="#">html</a>
+     </li>
+     <li>
+       <a class="Tags" href="#">Google</a>
+     </li>
+   </ul>
+   <button class="btn">See Project</button>
+  </div>
+  <div class="ProjectPage">
+    <h2 class="ProjectPage-h2">Professional Art<br>Printing Data</h2>
+   <p>A daily selection of privately personalized reads; no accounts
+      or sign-ups required. has been the industry's standard</p>
+   <ul class="project-ul">
+     <li>
+       <a class="Tags" href="#">Bootstrap</a>
+     </li>
+     <li>
+       <a class="Tags" href="#">html</a>
+     </li>
+     <li>
+       <a class="Tags" href="#">Google</a>
+     </li>
+   </ul>
+   <button class="btn">See Project</button>
+  </div>
+  <div class="ProjectPage">
+    <h2 class="ProjectPage-h2">Professional Art <br>Printing Data</h2>
+   <p>A daily selection of privately personalized reads; no accounts
+      or sign-ups required. has been the industry's standard</p>
+   <ul class="project-ul">
+     <li>
+       <a class="Tags" href="#">Bootstrap</a>
+     </li>
+     <li>
+       <a class="Tags" href="#">html</a>
+     </li>
+     <li>
+       <a class="Tags" href="#">Google</a>
+     </li>
+   </ul>
+   <button class="btn">See Project</button>
+  </div>
+  <div class="ProjectPage">
+   <div class="bg-img"></div>
+    <h2 class="ProjectPage-h2">Professional Art<br>Printing Data</h2>
+   <p>A daily selection of privately personalized reads; no accounts
+      or sign-ups required. has been the industry's standard</p>
+   <ul class="project-ul">
+     <li>
+       <a class="Tags" href="#">Bootstrap</a>
+     </li>
+     <li>
+       <a class="Tags" href="#">html</a>
+     </li>
+     <li>
+       <a class="Tags" href="#">Google</a>
+     </li>
+   </ul>
+   <button class="btn">See Project</button>
+  </div>
+  <div class="AboutMe headline">
+    <h1 class="headline-h1">About Me</h1>
+    <p class="AboutMe-p">Hello I’m a software developer! I can help you build a product ,
+      feature or website Look through some of my work and experience!
+      If you like what you see and have a project you need coded,
+      don’t hestiate to contact me.</p>
+    <button class="btn">Get My Resume</button>
+    <div class="underline"></div>
+    <ul class="Aboutme-list">
+      <li id="language">
+        <a class="AbotmeTab" href="#">Language</a>
+        <ul class="lang-list">
+          <li class="lang-sublist">
+            <a class="lang-tabs" href="#">Javascript</a>
+          </li>
+          <li class="lang-sublist">
+            <a class="lang-tabs" href="#">Ruby</a>
+          </li>
+          <li class="lang-sublist">
+            <a class="lang-tabs" href="#">Html</a>
+          </li>
+          <li class="lang-sublist">
+            <a class="lang-tabs" href="#">Css</a>
+          </li>
+        </ul>
+      </li>
+      <li id="frameworks">
+        <a class="AbotmeTab" href="#">Frameworks</a>
+        <ul class="lang-list">
```

Comment:
- [ ] I noticed here we are missing styling on this part, I doubted you didn't do because your code was clean before that 🙂, So I investigated I figured out you just had a typo in the class name `AbotmeTab` where it should be `AboutmeTab` could you please fix that :shipit: so our implementation will match the Figma design
![image](https://user-images.githubusercontent.com/87689547/153661264-3c3aaaff-72cf-4a26-9648-95b372ee600e.png)

Reply:
Thank you for your feedback.  There is typing mistake, I did not write the "u" 😄.  I also forgot to add @import link for special fonts. Now, I've fixed them.